src/assets/styles/_header.scss

```diff
@@ -30,5 +30,5 @@ h3.heading {
 }
 h4.heading,
 h5.heading {
-  @apply text--1h;
+  @apply text-n1h;
```

Comment:
https://github.com/tailwindlabs/tailwindcss/issues/6599

classes that start with a dash were not working. They would either not be applied or cause a build error.

I changed all of them to be n2h instead of -2h.

tailwindlabs/tailwindcss#6599